vumi/transports/vumi_bridge/vumi_bridge.py

```diff
@@ -323,10 +231,21 @@ def handle_raw_inbound_event(self, request):
             msg = TransportEvent(_process_fields=True, **to_kwargs(data))
             yield self.handle_inbound_event(msg)
             request.finish()
+            if msg.payload["event_type"] == "ack":
+                self.update_status(
+                    status='ok', component='sent-by-vumi-go',
+                    type='vumi_go_sent', message='Sent by Vumi Go')
+            else:
+                self.update_status(
+                    status='down', component='sent-by-vumi-go',
+                    type='vumi_go_failed', message='Vumi Go failed to send')
         except Exception as e:
             log.err(e)
             request.setResponseCode(400)
             request.finish()
+            self.update_status(
+                status='down', component='sent-by-vumi-go',
+                type='bad_request', message='Bad request sent')
```

Comment:
Maybe the type here should be `vumi_go_bad_event` and the message `Bad event received from Vumi Go`.